extensions/query-store/src/reports/overallResourceConsumption.ts

```diff
@@ -32,4 +34,10 @@ export class OverallResourceConsumption extends BaseQueryStoreReport {
 
 		return [durationContainer, executionCountContainer, cpuTimeContainer, logicalReadsContainer];
 	}
+
+	public override async configureButtonClick(configureDialog: ConfigureDialog): Promise<void> {
```

Comment:
If this doesn't get called outside of the report, can this be protected instead of public?